resources/user-agents/browsers/uc-browser/uc-web-10.json

```diff
@@ -439,6 +439,53 @@
             "iOS_A",
             "iOS_C"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*CriOS/*UCBrowser/#MAJORVER#.#MINORVER#*",
+          "engine": "WebKit",
+          "devices": {
+            "iPhone": "iPhone",
+            "iPod": "iPod Touch"
+          },
+          "properties": {
+            "Browser_Modus": "Chrome"
+          },
+          "platforms": [
+            "iOS_A_10_3", "iOS_A_10_2", "iOS_A_10_1", "iOS_A_10_0",
+            "iOS_A_9_3", "iOS_A_9_2", "iOS_A_9_1", "iOS_A_9_0",
+            "iOS_A_8_4", "iOS_A_8_3", "iOS_A_8_2", "iOS_A_8_1", "iOS_A_8_0", "iOS_A_8_0_B",
+            "iOS_A_7_1", "iOS_A_7_0",
+            "iOS_A"
+          ]
+        },
+        {
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*CriOS/*UCBrowser/#MAJORVER#.#MINORVER#*",
```

Comment:
I did test this on my phone. UC Browser has an option in the settings to change the useragent to be "Chrome":

![upload_thumb](https://cloud.githubusercontent.com/assets/136907/26119943/518b4e9e-3a2c-11e7-8da1-62cb0d0ccad6.png)